components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -235,8 +237,8 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
                 >
                     <span className='MenuItem__primary-text'>
                         {textProp}
-                        {selectedValueText && selectedValueElement}
-                        {id !== 'SidebarChannelMenu-moveToDivider' &&
+                        {renderSelected && selectedValueElement}
```

Comment:
What's this change for? 0/5 on it overall, but I'd assume that we just wouldn't pass `selectedValueText` if we don't want the selection to be rendered.

Reply:
we pass the selectedValueText because if it matched the value in the submenu it would display the checkmark icon but sometimes we don't want it in the parent menuitem as well :/